README.md

```diff
@@ -56,8 +53,7 @@ To watch for changes and continually rebuild the package (this is useful if you'
 npm run dev
 ```
 
-The compiler is written in [TypeScript](https://www.typescriptlang.org/), but don't let that put you off — it's basically just JavaScript with type annotations. You'll pick it up in no time. If you're using an editor other than [Visual Studio Code](https://code.visualstudio.com/), you may need to install a plugin in order to get syntax highlighting and code hints, etc.
-
+The compiler is written in [TypeScript](https://www.typescriptlang.org/), but don't let that put you off — it's just JavaScript with type annotations. You'll pick it up in no time. If you're using an editor other than [Visual Studio Code](https://code.visualstudio.com/), you may need to install a plugin to get syntax highlighting and code hints, etc.
```

Comment:
removing "basically" here seems slightly less accurate, but more importantly I'm not sure it's really worth our time optimizing the exact right wording to use